lib/trailblazer/developer/trace.rb

```diff
@@ -18,8 +18,13 @@ def call(activity, (ctx, flow_options), circuit_options={})
       alias_method :invoke, :call
 
       def arguments_for_call(activity, (options, flow_options), **circuit_options)
+        flow_options ||= {}
```

Comment:
That's needed because we need default tracing options to be merged into given `flow_options`. But it could be `nil`.

This has been replaced with splat merge instead `{ **default, **given }`